specification/msi/resource-manager/Microsoft.ManagedIdentity/preview/2015-08-31-preview/ManagedIdentity.json

```diff
@@ -343,22 +368,66 @@
         },
         "properties": {
           "x-ms-client-flatten": true,
+          "readOnly": true,
           "description": "The properties associated with the identity.",
           "$ref": "#/definitions/IdentityProperties"
         },
         "type": {
           "type": "string",
           "readOnly": true,
-          "enum": [
-            "Microsoft.ManagedIdentity/userAssignedIdentities"
-          ],
+          "enum": ["Microsoft.ManagedIdentity/userAssignedIdentities"],
           "x-ms-enum": {
             "name": "UserAssignedIdentities",
             "modelAsString": true
           },
           "description": "The type of resource i.e. Microsoft.ManagedIdentity/userAssignedIdentities"
         }
       },
+      "required": [
+        "location"
+      ],
+      "x-ms-azure-resource": true,
+      "description": "Describes an identity resource."
+    },
+    "SystemAssignedIdentity": {
+      "properties": {
+        "id": {
```

Comment:
These common properties (id, name, type, location, tags) are a common definition. It's best to use in a shared definition or optimally, linked from: https://github.com/Azure/azure-rest-api-specs/blob/fbc094f3097e954f008f8f415a2f95a6dfa3c13b/specification/common-types/resource-management/v1/types.json#L78